pkgs/development/tools/jo/default.nix

```diff
@@ -1,15 +1,14 @@
 {stdenv, fetchFromGitHub, autoreconfHook}:
 
 stdenv.mkDerivation rec {
-  name = "jo-${version}";
-  version = "1.1";
+  name    = "jo-${version}";
```

Comment:
hmm, I'm no big fan of this alignment.
1. It wastes more space
2. If we, for example, change name to pname, we either generate diff noise or leave it inconsistent.

Reply:
I will change it to pname right away and fix alignment.